nncf/compression_method_api.py

```diff
@@ -100,6 +104,9 @@ def state_dict(self):
     def initialize(self):
         pass
 
+    @property
+    def current_epoch(self):
```

Comment:
As we use `sparsity_init` to initialize algorithm directly and so we don't call set_sparsity in `CompressionScheduler` constructor it seems that we do not need `current_epoch` and `_current_epoch` anymore - `last_epoch` should be enough.

Reply:
@gadylshintr I think that thinking in `current_epoch` terms is more straightforward.

Reply:
Okay. It is probably should be the same change to `last_step` for consistency?

Reply:
I agree, @asenina please modify the code accordingly

Reply:
Done.